docs/src/user/variables-guide.rst

```diff
@@ -459,3 +459,129 @@ Workload import/export variables
 - ``os_migrate_workload_stop_before_migration`` - Set to true if you wish
   for os_migrate to stop your workloads/vms prior to migration. Note that
   only workloads/vms in `SHUTOFF` state will be migrated.
+
+Workload migration variables
+----------------------------
+Workloads to be migrated with OS Migrate can have varying storage configurations in
+the source cloud, and the desired way to migrate their storage also varies, per cloud
+operators preference.
+
+The following table summarizes the matrix of options (whats in the source, how
+it should be migrated, how should OS Migrate workloads YAML file be configured,
+is the conversion host required for this mode of migration, is this migration
+mode implemented).
+
+.. list-table:: Workload Migration options
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - Source workload
+     - Desired migration properties
+     - Migration parameter settings
+     - Conversion hosts required?
+   * - Booted from image (local boot disk), optionally additional volumes attached
+     - Boot disk not copied, VM booted from image (local boot disk), any additional volumes copied and attached
+     - _migration_params:
```

Comment:
These code blocks don't get rendered correctly, i wonder if we could do a block enclosed with '```' inside the table cell? But in general ++ on having the table here.

https://github.com/os-migrate/os-migrate/blob/2ad62a288fc11698b50472388d35b2d8fce4a701/docs/src/user/variables-guide.rst#workload-migration-variables

Reply:
I meant for specific elements in the table. But the code looks good and CI is green, so let's merge and fix up the docs in a follow up.